lagtraj/forcings/conversion/targets/dephy.py

```diff
@@ -539,15 +603,15 @@ def nudging_inv_time_prof(nudging_parameters, variable):
         "forc_omega": parameters.forc_omega,
         "forc_w": parameters.forc_w,
         "forc_geo": parameters.forc_geo,
-        "nudging_u": parameters.nudging_u,
-        "nudging_v": parameters.nudging_v,
-        "nudging_temp": parameters.nudging_temp,
-        "nudging_theta": parameters.nudging_theta,
-        "nudging_thetal": parameters.nudging_thetal,
-        "nudging_qv": parameters.nudging_qv,
-        "nudging_qt": parameters.nudging_qt,
-        "nudging_rv": parameters.nudging_rv,
-        "nudging_rt": parameters.nudging_rt,
+        "nudging_u": nudging_u,
```

Comment:
as above I think it would be clearer to have a list of the attributes to copy here, something like:

```python
nudged_vars = "u v temp theta ...".split()
attrs_to_copy = (
   [f"nudging_{var_name}" for var_name in nudged_vars]
   + ["forc_omega", ...]
)
```

and then do

```python
for attr_name in attrs_to_copy:
   dephy_dictionary[attr_name] = getattr(parameters, attr_name)
```

if you aren't getting the attributes from say `parameters` I would create a `dict` already above called `dephy_params` or something and use that to update `dephy_dictionary`